src/DatePicker/Calendar.js

```diff
@@ -47,7 +46,7 @@ class Calendar extends React.Component {
     open: PropTypes.bool,
     shouldDisableDate: PropTypes.func,
     showActionButtons: PropTypes.bool,
-    wordings: PropTypes.object,
+    wordings: deprecated(PropTypes.object, 'Instead, use `cancelLabel` and `okLabel`.'),
```

Comment:
You should only need the warning in DatePicker, otherwise the warning will fire multiple times as the value propagates down the component tree.